src/Utils/QueryAnalyzer.php

```diff
@@ -133,15 +144,18 @@ public static function is_enabled(): bool {
 	}
 
 	/**
-	 * @param \WPGraphQL\Request $request The GraphQL request being executed
+	 * Get the GraphQL Schema.
+	 * If the schema is not set, it will be set.
+	 *
+	 * @return \GraphQL\Type\Schema|\WPGraphQL\WPSchema|null
```

Comment:
I think the property type is wrong, and it/this function can return just `?\WPGraphQL\WPSchema`

Reply:
phpstan disagrees: 

```
Method WPGraphQL\Utils\QueryAnalyzer::get_schema() should return WPGraphQL\WPSchema but returns GraphQL\Type\Schema.
```

Reply:
Ah, found out why. Ya, I can get this fixed.